src/View/Helper/FormHelper.php

```diff
@@ -50,15 +50,21 @@ class FormHelper extends Helper
         'checkboxContainer' => '<div class="form-group form-check {{type}}{{required}}">{{content}}{{help}}</div>',
         'checkboxContainerError' => '<div class="form-group form-check {{type}}{{required}} is-invalid">{{content}}{{error}}{{help}}</div>',
         'checkboxFormGroup' => '{{input}}{{label}}',
-        'checkboxWrapper' => '<div class="form-check">{{input}}{{label}}</div>',
-        'radioWrapper' => '<div class="form-check">{{hidden}}{{input}}{{label}}</div>',
+        'checkboxWrapper' => '<div class="form-check">{{label}}</div>',
+        'checkboxInlineWrapper' => '<div class="form-check form-check-inline">{{label}}</div>',
+        'radioWrapper' => '<div class="form-check">{{hidden}}{{label}}</div>',
         'radioInlineFormGroup' => '{{label}}<div class="form-check form-check-inline">{{input}}</div>',
         'radioNestingLabel' => '<div class="form-check">{{hidden}}<label{{attrs}}>{{input}}{{text}}{{tooltip}}</label></div>',
         'staticControl' => '<p class="form-control-plaintext">{{content}}</p>',
         'inputGroupAddon' => '<div class="{{class}}">{{content}}</div>',
         'inputGroupContainer' => '<div{{attrs}}>{{prepend}}{{content}}{{append}}</div>',
         'inputGroupText' => '<span class="input-group-text">{{content}}</span>',
-        'file' => '<input type="file" class="form-control-file" name="{{name}}"{{attrs}}>'
+        'file' => '<input type="file" class="form-control-file" name="{{name}}"{{attrs}}>',
```

Comment:
Not related to this PR which only deals with multicheckbox but we need to fix the template for `file`. As seen in your demo app currently the input tag has the `class` attribute twice.

Reply:
Indeed. That will then also fix the missing error message.